src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,22 @@
-export const TodoList = () => {};
+import { TodoInfo } from '../TodoInfo';
+import { User } from '../../types/users';
+import { Todo } from '../../types/todo';
+
+type TodoListType = {
+  users:User[];
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<TodoListType> = ({ todos, users }) => {
+  return (
+    <section className="TodoList">
+      {todos.map(todo => {
+        const userTodos = users.find(user => user.id === todo.userId);
```

Comment:
What if user is not found?

Reply:
if the user is not found, then the block will not be added
![image](https://github.com/mate-academy/react_add-todo-form/assets/122548742/cb85ccdf-6bbc-4e00-8b3a-d778e43b4a5a)